memtrace.opam

```diff
@@ -10,7 +10,7 @@ homepage: "https://github.com/janestreet/memtrace"
 bug-reports: "https://github.com/janestreet/memtrace/issues"
 depends: [
   "dune" {>= "2.3"}
-  "ocaml" {>= "4.11.0"}
+  "ocaml" {>= "5.2.0" | <= "4.14"}
```

Comment:
I think 5.3 will be the first version with statmemprof? Not sure if there's a plan to backport before the 5.2 release.

Reply:
shouldn't it be `{(>= "4.11.0" & < "5") | >= "5.3.0"}`, or isn't this supposed to work with 4.x statmemprof?

Reply:
I expect this to work with 4.x statmemprof. I've only tested on 4.14 but the API has been stable since 4.11.